diff --git a/node_modules/@vueuse/core/index.mjs b/node_modules/@vueuse/core/index.mjs
index ab854f8..cc35e35 100644
--- a/node_modules/@vueuse/core/index.mjs
+++ b/node_modules/@vueuse/core/index.mjs
@@ -501,7 +501,7 @@ function useMounted() {
   if (instance) {
     onMounted(() => {
       isMounted.value = true;
-    }, isVue2 ? null : instance);
+    }, isVue2 ? undefined : instance);
   }
   return isMounted;
 }
